tests/specs/add/npm_jsr/__test__.jsonc

```diff
@@ -0,0 +1,21 @@
+{
+  "tempDir": true,
+  "tests": {
+    "npm": {
+      "steps": [
+        {
+          "args": "add --npm @denotest/esm-basic jsr:@denotest/add npm:chalk",
+          "output": "npm.out"
+        }
+      ]
+    },
+    "jsr": {
+      "steps": [
+        {
+          "args": "add --jsr npm:@denotest/esm-basic @denotest/add npm:chalk",
```

Comment:
Nit: Should we also add tests for these?

1. `deno install`
2. `deno add --npm npm:chalk` and `deno add --jsr jsr:@denotest/add`